libraries-http-2/src/main/java/com/baeldung/okhttp/ssl/TrustAllCertsClient.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.okhttp.ssl;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+public class TrustAllCertsClient {
+
+    public static OkHttpClient getTrustAllCertsClient() throws NoSuchAlgorithmException, KeyManagementException {
+        TrustManager[] trustAllCerts = new TrustManager[]{
+                new X509TrustManager() {
+                    @Override
+                    public void checkClientTrusted(java.security.cert.X509Certificate[] chain, String authType) {
+                    }
+
+                    @Override
+                    public void checkServerTrusted(java.security.cert.X509Certificate[] chain, String authType) {
+                    }
+
+                    @Override
+                    public java.security.cert.X509Certificate[] getAcceptedIssuers() {
+                        return new java.security.cert.X509Certificate[]{};
+                    }
+                }
```

Comment:
This block should begin only 4 (not 8) spaces indented from line 16.

Reply:
Thank you, Kevin Gilmore.
I fixed it.